spec/util/postgres_admin_spec.rb

```diff
@@ -1,6 +1,40 @@
 require "util/postgres_admin"
 
 describe PostgresAdmin do
+  describe ".restore", :if => RSpec.configuration.with_postgres_specs do
+    context "with a pg_dump file" do
+      include_context "Database Restore Validation Helpers"
+      let(:dbname) { "pg_dump_restore_of_simple_db" }
+
+      it "restores all of the tables to the new database name" do
+        restore_opts = RestoreHelper.default_restore_dump_opts.dup
+        restore_opts[:dbname] = "pg_dump_restore_of_simple_db"
+        PostgresAdmin.restore(restore_opts)
+
+        expect(author_count).to eq(2)
+        expect(book_count).to   eq(3)
+      end
+    end
+
+    context "with a pg_basebackup file" do
+      include_context "Database Restore Validation Helpers"
```

Comment:
Could we move this to the `describe` level?

Reply:
Probably, I will look into that.